### ISSUE TITLE
Error: invalid csrf token in maintenance mod.

### ISSUE THREAD

**Issue by Lazerian** [NodeBB/NodeBB#2662]
When I put the forum in maintenance mode, I can not login.
I am redirected to a page where it's just written "Forbidden".

I use mongodb and defaults settings.

error log:
```
[11451] - error: /login
 Error: invalid csrf token
    at verifytoken (/path/to/nodebb/node_modules/csurf/index.js:237:11)
    at csrf (/path/to/nodebb/node_modules/csurf/index.js:100:7)
    at Layer.handle [as handle_request] (/path/to/nodebb/node_modules/express/lib/router/layer.js:82:5)
    at next (/path/to/nodebb/node_modules/express/lib/router/route.js:100:13)
    at Route.dispatch (/path/to/nodebb/node_modules/express/lib/router/route.js:81:3)
    at Layer.handle [as handle_request] (/path/to/nodebb/node_modules/express/lib/router/layer.js:82:5)
    at /path/to/nodebb/node_modules/express/lib/router/index.js:235:24
    at Function.proto.process_params (/path/to/nodebb/node_modules/express/lib/router/index.js:313:12)
    at /path/to/nodebb/node_modules/express/lib/router/index.js:229:12
    at Function.match_layer (/path/to/nodebb/node_modules/express/lib/router/index.js:296:3)
```

The only solution I found is to drop the database and setup nodebb.

**Comment by julianlam** [comment]
Latest code should not have this problem, so 0.6.1, when launched

**Comment by Lazerian** [comment]
I confirm, the problem is solved with v0.6.1-dev release.

Thank you for your help. I am discovering NodeBB, and it is really a brilliant forum software!

**Comment by psychobunny** [comment]
thanks :)
